Update intents.md (#1505)
discordjs/guide: guide/popular-topics/intents.md

To specify which events you want your bot to receive, first think about which events your bot needs to operate. Then select the required intents and add them to your client constructor, as shown below.

> All gateway intents, and the events belonging to each, are listed on the [Discord API documentation](https://discord.com/developers/docs/topics/gateway#list-of-intents).
> 
> The library equivalent of which are listed on the [Discord API Types](https://discord-api-types.dev/api/discord-api-types-v10/enum/GatewayIntentBits).

Reviewer: ```suggestion
All gateway intents, and the events belonging to each, are listed on the [Discord API documentation](https://discord.com/developers/docs/topics/gateway#list-of-intents). You can also find the enum values used in discord.js [here](https://discord-api-types.dev/api/discord-api-types-v10/enum/GatewayIntentBits).
```

> To see their discord.js-specific names, see them listed on the [Discord API Types](https://discord-api-types.dev/api/discord-api-types-v10/enum/GatewayIntentBits).

Reviewer 2: ```suggestion
The library equivalent of which are listed on the [Discord API Types](https://discord-api-types.dev/api/discord-api-types-v10/enum/GatewayIntentBits).
```
discord.js-specific name doesn't seem correct, perhaps something like this would be better?

also maybe there should be some indication that discord api types is a package here different from discord api docs? seems like something that's easy to confuse

Author: I agree that the wording you provided is much better.

I originally suggested the addition just so that people would be able to find the library names without having to search too much, and this clarifies that.
